telebot/types.py

```diff
@@ -6428,15 +6427,20 @@ def de_json(cls, json_string):
             obj['thumbnail'] = None
         return cls(**obj)
 
-    def __init__(self, name, title, sticker_type, is_animated, is_video, stickers, thumbnail=None, **kwargs):
+    def __init__(self, name, title, sticker_type, is_animated=None, is_video=None, stickers=None, thumbnail=None, **kwargs):
         self.name: str = name
         self.title: str = title
         self.sticker_type: str = sticker_type
-        self.is_animated: bool = is_animated
-        self.is_video: bool = is_video
         self.stickers: List[Sticker] = stickers
         self.thumbnail: PhotoSize = thumbnail
 
+        if is_animated is not None:
```

Comment:
They will always be None from now on, meaning that they will always get deprecation warnings.
Instead, I propose turning them into property and showing deprecation warning?

Reply:
None will not trigger the warning. NOT NONE will: "NOT NONE" means user use old-style call signature.

Reply:
hmm true, wasn't attentive
Anyways, since the class represents an object from API, is_animated and is_video parameters are not passed. So my suggestion about totally removing these params but making them remain as properties with deprecation warning is good?

Reply:
I did not checked if the class is created by user or not. If it is just deserializable - this is the right way. I'll update.

Reply:
@coder2020official both properties will return TRUE?

Reply:
Both properties will return FALSE?